packages/next-on-pages/templates/_worker.js/utils/routing.ts

```diff
@@ -6,10 +6,16 @@ import {
 	applySearchParams,
 } from './http';
 
+export type MatchedSetHeaders = {
+	normal: Headers;
+	important: Headers;
+	middlewareLocation?: string | null;
```

Comment:
could you add a comment explaining that this is not standard and why (and how) we need to use it? 🙏